faunadb/json.go

```diff
@@ -160,6 +160,12 @@ func (p *jsonParser) parseQuery() (value Value, err error) {
 		value = QueryV{lambda}
 	}
 
+	if token, err := p.decoder.Token(); err == nil {
```

Comment:
You can't use `:=`. That will create a new `err` variable. Therefore, the current implementation will return `nil` in case the token isn't `}`, instead of the `wrongToken` error.

Reply:
Perhaps we should create a test case for this failure, if possible.